database.sql

```diff
@@ -786,6 +786,7 @@ CREATE TABLE IF NOT EXISTS `item-content` (
 	`verb` varchar(100) NOT NULL DEFAULT '' COMMENT 'ActivityStreams verb',
 	 PRIMARY KEY(`id`),
 	 UNIQUE INDEX `uri-plink-hash` (`uri-plink-hash`),
+	 FULLTEXT INDEX `body` (`body`),
```

Comment:
I believe this index needs needs to cover `title`, `content-warning`, `location` and `app` as well as they all are displayed and may be expected to show up in a search.

Reply:
I don't think that `location` and `app` should be used for the search. But `title` and `content-warning` sounds okay.

Reply:
Why not? They are user-controlled public fields. What if someone wants to retrieve all posts submitted through Tusky? What if someone wants to retrieve all posts reportedly submitted from Germany?

Reply:
It will also lead to (i guess massive) false positives. When you - for example - are searching for content about "android" - but you also get all posts that had been posted via some app that announces itself as "... for android" - or you search for something that happened in hamburg - but you get all posts that had been posted from hamburg as well.

Reply:
Fair enough. Title and content warning still should be part of this index and the associated queries.

Reply:
Done